app/models/invoice.rb

```diff
@@ -11,6 +11,15 @@ def self.incomplete_invoices
     joins(:invoice_items).where('invoice_items.status != ?', 2).distinct
   end
 
+  def creation_date_formatted
```

Comment:
Yeah I really like this being in a method - we can go back later and reuse this in a bunch of places I believe